Src/Common.ps1

```diff
@@ -343,11 +343,15 @@ function RemoveBatchVMs {
     .PARAMETER credentialsKind
         Mandatory. Type of credential. Accepted values are "File" or "RunBook"
 
+    .PARAMETER profilePath
+        Optional. Path to file with Azure Profile.
+        Default "$env:APPDATA\AzProfile.txt".
+
     .NOTES
 
     #>
     [CmdletBinding()]
-    param($vms, $BatchSize, $credentialsKind)
+    param($vms, $BatchSize, $credentialsKind, $profilePath)
```

Comment:
But then profilePath is not used i the function (i.e. it is not passed to Remove-AzureDtlLabVMs. If we reintroduce this parameter, we have to understand why it was removed, if it is ok to put it back and test it at least once.